include/mbedtls/ecp.h

```diff
@@ -128,6 +129,27 @@ typedef enum
     MBEDTLS_ECP_DP_CURVE448,       /*!< Domain parameters for Curve448. */
 } mbedtls_ecp_group_id;
 
+/*
+ * Curve NamedGroup IDs
+ */
+typedef enum
+{
+    MBEDTLS_ECP_DP_SECP192K1_TLS_ID = 18,
```

Comment:
I think those constants should be declared as macros, like other constants from the standard. Moreover, they should not belong to the ECP namespace, because that will go away with the takeover of PSA.

Reply:
I would suggest `#define MBEDTLS_TLS13_NAMED_GROUP_XXX`. 

@ronald-cron-arm WDYT?

Reply:
Amended

Reply:
Amended